proto/xudrpc.proto

```diff
@@ -77,7 +77,7 @@ service Xud {
     };
   }
   
-  /* Subscribe to incoming peer orders. */
+  /* Subscribe to incoming peer orders and cancellations. */
```

Comment:
I also think I prefer "Subscribe to peer order events" here. I think eventually it would be good to add more detailed comments to our proto file to clearly define how each call works, and there we can elaborate on exactly which events are covered like new incoming orders, orders being canceled, orders being filled between other peers, and how orders can be only partially filled or invalidated. But for a one-liner that's my preference.